Dispatcher/UI/Homepage/HomepageViewController.swift

```diff
@@ -129,12 +144,20 @@ extension HomepageViewController: UITableViewDataSource {
     }
 }
 
-extension HomepageViewController: ArticleCellDelegate, UITableViewDelegate {
+extension HomepageViewController: ArticleCellDelegate, UITableViewDelegate, HeaderViewDelegate {
+    func searchPressed() {
+        self.performSegue(withIdentifier: SegueIdentifiers.fromHomepageToSearchScreen, sender: self)
+    }
+    
+    func notificationsPressed() {
+        
+    }
+    
     func navigateButtonPressed(_ articleID: String) {
-        print(articleID)
+        
     }
     
     func favoritesButtonPressed(_ articleID: String) {
```

Comment:
We would prefer not to leave empty functions laying around in our codebase as this can confuse future developers - is there logic missing? Is this function redundant?